.github/workflows/ci.yml

```diff
@@ -9,8 +9,6 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [
-          2.7,
-          3.5,
```

Comment:
Python 3.5 was reached end-of-life last year. 
- https://www.python.org/downloads/release/python-3510/

Reply:
Let's make a separate PR to remove 3.5 and merge it faster. I see no reason not to.

Reply:
Ok, reverted.